hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -392,6 +392,38 @@ def calc_examples(self):
     return examples
 
 
+class DataObserver(object):
+    """Observer class for recording the behaviour of a
+    ConjectureData object, primarily used for tracking
+    the behaviour in the tree cache."""
+
+    def init(self, data):
+        """Init method to be called with the actual
+        data object at end of __init__.
+
+        FIXME: To be removed during this PR. Only here
```

Comment:
Is this hook supposed to have been removed by now?

Reply:
It is! Whoops.